### DISCUSSION TITLE
add ":f" as an alias for ":fmt/:format"

### DISCUSSION THREAD

**Discussion by samankittani** [helix-editor/helix#11621] [Ideas]
:f isn't used for anything, and ":fmt" is a bit cumbersome. This would be better suited as a custom command, but that isn't possible yet (#4423). It would be nice to have ":fw" and ":fwq" to format-write and format-write-quit.

**Comment by Rudxain** [comment]
I'm unsure if this is worth it:
- If a user has `auto-format = true` in their `languages.toml`, then they wouldn't need the cmd (unless they want to reduce writes to storage).
- If `auto-format = false` for some lang, then they only want to `fmt` some (or none) files. In that case, they can simply `:set-option auto-format = true` whenever they open a file they want to `fmt`.
- If a user really dislikes `auto-format` (maybe because it's distracting?), then they would rarely ever `fmt` manually. Maybe they use a tool that formats all files in a repo before `commit`, rather than `fmt` while the editor is open.

However, those 3 points assume the buffer is _associated with a file_.

Sometimes, it makes sense to view/edit a buffer without ever writing it to the file-system. Pasting into a buffer is one of those cases.

**Comment by chtenb** [comment]
There are also many key combinations available. You could e.g. consider assigning a function key to an action that you do frequently.
